specs/altair/fork.md

```diff
@@ -17,7 +17,7 @@
 
 ## Introduction
 
-This document describes the process of the first upgrade of Ethereum 2.0: the Altair hard fork, introducing light client support and other improvements.
+This document describes the process of the first upgrade of the eacon chain: the Altair hard fork, introducing light client support and other improvements.
```

Comment:
```suggestion
This document describes the process of the first upgrade of the beacon chain: the Altair hard fork, introducing light client support and other improvements.
```

Reply:
surprised this wasn't caught by spell check. looking to see if altair/ is being checked